tuxemon/states/world/worldstate.py

```diff
@@ -293,6 +296,13 @@ def set_layer(self) -> None:
         self.layer.fill(self.layer_color)
         self.screen.blit(self.layer, (0, 0))
 
+    def set_bubble(self) -> None:
+        if self.bubble:
+            for npc, surface in self.bubble.items():
+                cx, cy = self.get_pos_from_tilepos(npc.tile_pos)
```

Comment:
these should be sorted by coordinates value so that they overlap in a predictable way.  if not sorted, they will flicker